src/courseware/course/sidebar/sidebars/outline/OutlineSidebar.test.jsx

```diff
@@ -2,11 +2,34 @@ import React from 'react';
 import {
   initializeMockApp, initializeTestStore, render, screen,
 } from 'setupTest';
+import { getConfig } from '@edx/frontend-platform';
 import SidebarContext from 'courseware/course/sidebar/SidebarContext';
-import OutlineSidebar from 'courseware/course/sidebar/sidebars/outline/OutlineSidebar';
+import OutlineSidebar, { postEventOutlineToSidebar } from 'courseware/course/sidebar/sidebars/outline/OutlineSidebar';
 
 initializeMockApp();
 
+describe('postEventOutlineToSidebar', () => {
+  it('post the event to the OutlineSidebar iframe', () => {
+    const iframe = {
+      contentWindow: {
+        postMessage: jest.fn(),
+      },
+    };
+    document.getElementById = jest.fn(() => iframe);
+
+    const message = { message: 'outline_event' };
+    const messageTargets = [getConfig().SIDEBAR_MFE_BASE_URL];
+
+    postEventOutlineToSidebar();
+
+    // Assert that the iframe's contentWindow.postMessage function was called with the correct arguments
+    expect(iframe.contentWindow.postMessage).toHaveBeenCalledWith(
+      message,
+      messageTargets[0],
+    );
+  });
+});
+
```

Comment:
If comment related to moving the function to the utils file, please move this to a new file utils.test.js